public/v4/apps/restyaboard.yml

```diff
@@ -0,0 +1,47 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: restyaplatform/restyaboard:dev
```

Comment:
`:dev` seems to be a dynamic tag. Please use a fixed tag. Looks like the latest is `v0.6.9`